web/_tool/common.dart

```diff
@@ -36,3 +36,16 @@ Future<bool> run(
 void logWrapped(int code, String message) {
   print('\x1B[${code}m$message\x1B[0m');
 }
+
+Iterable<String> listPackageDirs(Directory dir) sync* {
```

Comment:
In all seriousness, how happy were you to find a spot to use a synchronous generator? 😄

Reply:
so stoked.